README.md

```diff
@@ -81,3 +78,89 @@ $> raito run
+| `sf-skip-tags`                              | If set, tags will not be fetched                                                                                                                                                                                                                                                                                                                                                                                                                | False     | `false`              |
+| `sf-skip-columns`                           | If set, columns and column masking policies will not be imported.                                                                                                                                                                                                                                                                                                                                                                               | False     | `false`              |
+| `sf-data-usage-window`                      | The maximum number of days of usage data to retrieve. Maximum is 90 days.                                                                                                                                                                                                                                                                                                                                                                       | False     | `90`                 |
+| `sf-database-roles`                         | If set, database-roles for all databases will be fetched.                                                                                                                                                                                                                                                                                                                                                                                       | False     | `false`              |
+
+## Supported features
+
+| Feature             | Supported | Remarks                                |
+|---------------------|-----------|----------------------------------------|
+| Row level filtering | ✅         | Only supported for enterprise editions |
+| Column masking      | ✅         | Only supported for enterprise editions |
+| Locking             | ✅         | Support for both who and what lock     |
+| Replay              | ✅         | Explicit deletes cannot be replayed    |
+| Usage               | ✅         |                                        |
```

Comment:
Are there limits to the usage parsing that we can mention?

Reply:
I don't think so. In addition with the full query, Snowflake is providing us with all used data objects, user, role, status, ...
The only thing that can be wrong is taht we associated to many permissions to a data object. For example when executing an update query with a join to another table. We will mark `UPDATE` to both DOs. 
I will update the docs